docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,264 @@
+= Policy Configuration
+
+When `ec` command line requires configuration, e.g. for `ec validate image`
+command, the configuration can be provided as an inline JSON or YAML string,
+such as in this example:
+
+[,bash]
+----
+ec validate image --policy '{
+    "configuration": {
+        "collections": ["minimal"]
+    }
+    "sources": [
+        {
+            "policy": ["oci::https://quay.io/hacbs-contract/ec-release-policy:latest"],
+            "data": ["oci::https://quay.io/hacbs-contract/ec-policy-data:latest"]
+        }
+    ]
+} ...'
+----
+
+As a reference to a xref:ecc:ROOT:reference.adoc[EnterpriseContractPolicy]
```

Comment:
I think this sentence needs to start with something like `A configuration can also be a reference ...`.